redbadger/website-honestly
# Read more slices for FM digital transformation and Sky case studies

### Motivation

The Fortnum & Mason case study should always link to the new Fortnum & Mason digital transformation case study. Similarly the Sky CMS case study should always link to the Sky case study.

This adds "read more" slices and static links between the mentioned case studies.

### Test plan

- Open "Fortnum & Mason" case study. Reload the page a few times. Make sure the first related case study at the bottom is always "Fortnum & Mason digitial transformation".
- Open "Sky CMS" case study. Reload the page a few times. Make sure the first related case study at the bottom is always "Sky".

### Pre-merge checklist
